src/dmd/globals.d

```diff
@@ -118,6 +118,14 @@ enum CxxHeaderMode : uint
     verbose /// Generate headers and add comments for hidden declarations
 }
 
+/// Trivalent boolean to represent the state of a `revert`able change
+enum RevertState
+{
```

Comment:
Can we use `byte` or similar as type to reduce the memory consumption?

Reply:
Done